webapp/src/shared/urls.ts

```diff
@@ -17,7 +17,7 @@ export const CHATROOM_URL = '/chatroom';
 export const CREATE_CHATROOM_URL = `${CHATROOM_URL}/createchatroom`;
 export const EDIT_CHATROOM_AVATAR_URL = `${CHATROOM_URL}/:chatroomId/edit/avatar`;
 export const COMPONENTS_BOOK_URL = '/components-book';
-export const DEFAULT_LOGIN_REDIRECT_URL = `${USERS_URL}`;
+export const DEFAULT_LOGIN_REDIRECT_URL = `${FRIENDS_URL}`;
```

Comment:
@Damrod you mean this url? I think is already changed. Or maybe you meant another thing?

Reply:
Yes, you have changed that one. I don't know why, but after login in I'm still redirected to the old users page.

Reply:
Totally right, we were using the user_page url instead of the default_login_redirect one. Now should be working. Thanks!

Reply:
Thanks!